docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,244 @@
+= Mobile Core (MC) Architecture Overview
+:toc:
+
+.Target Audience
+****
+The target audience for this document is developers who want a better understanding of what Mobile Core does, what value it brings and if they were to build a Mobile Service, how would they get MC to discover it and integrate with it.
+****
+
+== Introduction
+The overall goal of a full Mobile solution is a group of services working together in order to solve a problem. Each of those component services offers a piece of the functionality necessary for the solution to work. But the overall solution needs all these component parts (i.e. services) to be integrated together. MC is the component which glues/integrates/orchestrates these componet services. MC is the orchestrator in the context of Mobile.
+
+As well as the integration of Mobile Services with each other, another key aspect of building a solution is the integration with the platform, which is OpenShift. MC takes ownership of this aspect of the solution design also.
+
+== Design Principles
+
+MC interacts with OpenShift and the Kubernetes Core and not directly with the Mobile Services.
+
+== MC Installation
+The MC makes the Openshift platform Mobile aware. To do this the MC must perform a number of actions as part of its installation process.
+
+
+* *Step 1*: Ensure that Mobile Specific Services are made available through the Service Catalog
+
+For mobile developers to be able to consume Mobile value add services, they must be available to them. The medium through which services are presented is the https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/index.html[Openshift Service Catalog]. In order to achieve this, the MC must perform a number of tasks:
+
+** *Step 1.1*: Install / Configure an Ansible Service Broker
```

Comment:
Since 3.7 the name is [OpenShift Ansible Broker](https://docs.openshift.com/container-platform/3.7/architecture/service_catalog/ansible_service_broker.html)